static/js/browser2/arc_section.tsx

```diff
@@ -64,12 +66,14 @@ export class ArcSection extends React.Component<
           labels={this.state.outLabels}
           provDomain={this.state.provDomain}
         />
-        <InArcSection
-          nodeName={this.props.nodeName}
-          dcid={this.props.dcid}
-          labels={this.state.inLabels}
-          provDomain={this.state.provDomain}
-        />
+        {this.props.pageDisplayType !== PageDisplayType.PLACE_STAT_VAR ? (
```

Comment:
It might be better to pass this boolean from the parent. This way this component is standalone (does not depend on parent component logic)

Reply:
done